cpp/README.md

```diff
@@ -5,36 +5,34 @@
 * cmake version: 3.18+
 ## Installation and Running TorchServe CPP
 
+This installation instruction assumes that TorchServe is already installed through pip/conda/source. If this is not the case install it after the `Install dependencies` step through your preferred method.
+
 ### Install dependencies
 ```
 cd serve
 python ts_scripts/install_dependencies.py --cpp --environment dev [--cuda=cu121|cu118]
 ```
 ### Building the backend
+Don't forget to install or update TorchServe at this point if it wasn't previously installed.
 ```
 ## Dev Build
 cd cpp
 ./build.sh [-g cu121|cu118]
 
-## Install TorchServe from source
-cd ..
-python ts_scripts/install_from_src.py
-```
-### Set Environment Var
-#### On Mac
-```
-export DYLD_LIBRARY_PATH=$DYLD_LIBRARY_PATH:$(pwd)/_build/_deps/libtorch/lib
-```
-#### On Ubuntu
-```
-export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$(pwd)/_build/_deps/libtorch/lib
 ```
 
 ### Run TorchServe
 ```
 mkdir model_store
 torchserve --ncs --start --model-store model_store
 ```
+
+### Clean the build directory
+To clean the build directory in order to rebuild from scratch simply delete the cpp/_build directory with
+```
+rm -rf cpp/_build
+```
+
```

Comment:
can we keep this so that cx can clean up env in case of debugging?

Reply:
@lxning Not sure if I understand. This is meant to be executed if you want to rebuild from scratch. Like a `make clean`. Can make this more clear in the description.

Reply:
We haven't set clean rule at this moment. We need let cx know how to clean.